### DISCUSSION TITLE
Do we  need to backport the sof patches to sof-github which sent directly to broonie git without  creating PR in sof-github

### DISCUSSION THREAD

**Discussion by Venkata-Prasad-Potturu** [thesofproject/sof#8794] [Q&A]
Hi @plbossart 
Below are the sof patches sent directly to mark brown's git without creating PR in SOF-Github.
**Do we need to backport these patches and create a PR in SOF-Github?**

ASoC: amd: acp: Drop redundant initialization of machine driver data	
ASoC: amd: acp: Make use of existing *_CODEC_DAI macros
ASoC: amd: acp: Add missing error handling in sof-mach
ASoC: amd: acp: Update MODULE_DESCRIPTION for sof-mach
ASoC: SOF: amd: Fix memory leak in amd_sof_acp_probe()
ASoC: SOF: amd: Optimize quirk for Valve Galileo
ASoC: SOF: core: Skip firmware test for custom loaders
ASoC: SOF: amd: Compute file paths on firmware load

https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=f0f1021fc9cb88ebdc241b6121107399ee4f2eb7
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=68ab29426d88294d16170919a6a6e764f375113f
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=d0ada20279db2649a7549a2b8a4a3379c59f238d
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=a4832a94688000662d4ebb8a1c05f086a9c98826
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=222be59e5eed1554119294edc743ee548c2371d0
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=a13f0c3c0e8fb3e61fbfd99c6b350cf9be0c4660
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=369b997a1371aeecd0a1fb0f9f4ef3747a1d07a4
https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=d9cacc1a2af2e1cd781b5cd2a3e57fbde64f5a2d

**Comment by marc-hb** [comment]
> Do we need to backport these patches and create a PR in SOF-Github?

I believe it's not required unless _you_ need them for some reason of yours (e.g.: validation, dependency,...). Longer explanation:

https://thesofproject.github.io/latest/contribute/linux/development_tree.html

**Reply by kv2019i** [reply]
@Venkata-Prasad-Potturu Let me ping @ujfalusi @bardliao @ranj063 and @dbaluta as well. And FYI to @plbossart as well.
 
I'll echo @marc-hb 's response that it's not mandatory. When sof-dev is rebased to upstream next time, the patches sent directly to upstream will be picked up then.

In practise, it is good to consider sending a backport to sof-dev (as a PR) especially if the changes are bigger and touch SOF core. This will help the maintainers job when doing the next rebase to usptream for github "sof-dev" and "sof-dev-rebase" branches.
